docs/_posts/2018-07-18-523-release.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: blogpost
+title:  "5.2.3 Release Announcement"
+section_id: blog
+date:   2018-07-18 12:00:00
+author: Open GEE Development Team
+---
+
+Hello Open GEE Community!
+ 
+We are excited to announce the official release of Open GEE 5.2.3!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+ 
+Enhancements include:
+
+**Open GEE C++ code now supports and defaults to compiling using the <code>-std=gnu++11</code> compile flag**. This default can be overridden and the C++ code can be compiled using the <code>-std=gnu++98</code> flag instead. Starting Open GEE 5.2.4 <code>-std=gnu++98</code> will be deprecated and will not be supported.
+
+**Remove TCP/IP connections to PostgreSQL.** Open GEE Server, by default, connects to PostgreSQL exclusively using Unix domain sockets. This makes it easier to secure PostgreSQL for security sensitive installations.
+
+**Volume deletion.** <code>geconfigureassetroot</code> now supports the option <code>--removevolume</code> for volume deletion.
+
+**Performance tuning for gecombineterrain.** <code>gecombineterrain</code> now supports the option <code>--numCompressThreads</code> which specifies the number of maximum worker threads used to compress packets in this operation.
+
+**Package name customization at build time.** Scons build now accepts a custom label argument which is appended to the version in RPM file names.
+
+**Rewrite KML URLs.** Users can rewrite the URLs for KML resources included in databases at publish time.
+
+**Maps API Javascript Files.** Maps API Javascript files are now available and installed at <code>/opt/google/gehttpd/htdocs/maps/mapfiles</code>.
+
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.2.3-4.final). The full release notes can also be found [here](http://www.opengee.org/geedocs/answer/7160003.html).
```

Comment:
release page link doesn't work, but that might be because it isn't live yet

Reply:
correct. it will be before we merge 5.2.3 branch into master.

Reply:
It's a catch 22. This PR has to be merged before we can add the final tag :)